src/style.css

```diff
@@ -1,114 +1,155 @@
-  border: 1px solid;
-}
-
-#add-form label {
-  margin-right: 10px;
-  border: 1px solid;
-}
-
-h2 {
-  font-size: smaller;
-}
-
-h3 {
-  font-size: smaller;
-}
-
-#score-btn {
-  display: flex;
-  flex-wrap: wrap;
-  align-items: center;
-  margin-left: 4.5rem;
-  gap: 3rem;
-}
-
-h4 {
-  font-size: x-large;
-}
-
-#forma {
-  display: flex;
-  flex-direction: column;
-  margin-top: -6.3rem;
-}
-
-#list-form-container {
-  display: flex;
-}
-
-h5 {
-  font-size: x-large;
-}
-
-h6 {
-  font-size: large;
-  padding: 5px;
-  border: 2px solid black;
-}
-
-#footer {
-  margin-top: auto;
-}
+body {
+  font-family: Arial, sans-serif;
+  padding: 0;
+  display: flex;
+  margin: 0;
+  min-height: 100vh;
+  height: min-content;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-image: url(./img/black-banner-yellow-light-background_33869-664.jpg);
+}
+
+#book-collection {
+  text-align: center;
+}
+
+h1 {
+  display: flex;
+  padding-left: 35px;
+  font-size: xx-large;
+  border: 2px solid #ff1825;
+  color: wheat;
+}
+
+#scores-list {
+  width: 83%;
+  height: 166px;
+  border-radius: 4px;
+  margin: 0;
+  overflow-y: scroll;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+}
+
+#scores-list::-webkit-scrollbar {
+  width: 0;
+  background-color: transparent;
+}
+
+#scores-list::-moz-scrollbar {
+  width: 0;
+  background-color: transparent;
+}
+
+li {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  align-content: center;
+  padding: 3px;
+  background-color: wheat;
+}
+
+li:nth-child(even) {
+  background-color: lightgray;
+}
+
+#add-form {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  gap: 1rem;
+  margin-top: -7rem;
+}
+
+#add-button {
+  border-radius: 8px;
+  border: 1px solid #060606;
+  width: 7rem;
+}
+
+#name {
+  border-radius: 8px;
+  border: 1px solid #060606;
+  height: 1.8rem;
+}
+
+#score {
+  border-radius: 8px;
+  border: 1px solid #060606;
+  height: 1.8rem;
+}
+
+#add-form label {
+  margin-right: 10px;
+  border: 1px solid;
+}
+
+h2 {
+  font-size: smaller;
+}
+
+h3 {
+  font-size: smaller;
+}
+
+#score-btn {
+  display: flex;
+  gap: 3rem;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+h4 {
+  font-size: x-large;
+  color: wheat;
+}
+
+#forma {
+  display: flex;
+  gap: 6rem;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+  align-items: center;
+  margin-top: -28px;
+}
+
+#list-form-container {
+  display: flex;
+  flex-direction: column;
+  margin-left: 3rem;
+  align-items: center;
+}
+
+h5 {
+  font-size: x-large;
+  color: wheat;
+}
+
+h6 {
+  font-size: large;
+  padding: 5px;
+  color: wheat;
+  border: 2px solid #ff1825;
+}
+
+#footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+}
+
+.refresh {
+  border-radius: 5px;
+  border: 1px solid #060606;
+  text-align: center;
+  padding: 5px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+}
```

Comment:
- [ ] Wonderful job so far. However, it is required to follow the general layout of the wireframe given in the project requirements. You're allowed to be dynamic in your styling but the general layout of the wireframe should be remained. Kindly follow the general layout specified for this project in the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png).

![general-layout](https://user-images.githubusercontent.com/26597432/217506643-f998a340-8120-4270-86a6-2db86e4d46e7.png)